pkgs/tools/typesetting/tex/texlive/combine.nix

```
@@ -122,23 +122,35 @@ in (buildEnv {
   '' +
     # now filter hyphenation patterns, in a hacky way ATM
   (let
-    pnames = uniqueStrings (map (p: p.pname) pkgList.splitBin.wrong);
+    hyphens = lib.filter (p: p.hasHyphens or false) pkgList.splitBin.wrong;
```

```suggestion
    hyphens = lib.filter (p: p.hasHyphens or false && p.tlType == "run") pkgList.splitBin.wrong;
```
(in reply to https://github.com/NixOS/nixpkgs/pull/127489/files#r663332781)

---

Squashed and pushed.